solarforecastarbiter/reports/templates/metrics_month.md

```diff
@@ -0,0 +1,8 @@
+
+### Monthly analysis
+
+Metrics for each month of the analysis period are displayed in tables and figures below. Starting with January (1) up to December (12).
```

Comment:
probably don't need the numbers now that you've added the nice calendar lookups

Reply:
Updated